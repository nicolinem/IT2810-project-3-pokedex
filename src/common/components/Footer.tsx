import programming from "../../public/assets/programming.png";

const Footer: React.FC = () => {
  return (
    <footer className="bg-[#121A36] text-white p-4 shadow md:px-6 md:py-8 bg-skin-secondary">
      <div className="sm:flex sm:items-center sm:justify-between">
        <a className="flex items-center mb-4 sm:mb-0">
          <img src={programming} className="h-8 mr-3" alt="Logo" />
          <span className="self-center text-2xl font-semibold whitespace-nowrap text-skin-primary">
            Team 1
          </span>
        </a>
        <ul className="flex flex-wrap items-center mb-6 text-sm sm:mb-0 text-skin-primary">
          <li>
            <a
              href="https://www.ntnu.no/studier/emner/IT2810#tab=omEmnet"
              className="mr-4 hover:underline md:mr-6 "
            >
              About
            </a>
          </li>
          <li>
            <a
              href="https://gitlab.stud.idi.ntnu.no/it2810-h22/Team-1/project-3"
              className="mr-4 hover:underline md:mr-6 "
            >
              GitLab
            </a>
          </li>
          <li>
            <a href="https://www.ntnu.no/idi" className="hover:underline">
              Contact
            </a>
          </li>
        </ul>
      </div>
      <hr className="my-6 text-skin-primary sm:mx-auto lg:my-8" />
      <span className="block text-sm sm:text-center text-skin-primary">
        © 2022
        <p> </p>
        <a href="https://www.ntnu.no/" className="hover:underline">
          Women in Tech NTNU™
        </a>
        . All Rights Reserved.
      </span>
    </footer>
  );
};

export default Footer;