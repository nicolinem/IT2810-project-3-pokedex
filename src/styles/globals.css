

.TypeButton {
    position: absolute;
    width: 90px;
    height: 27px;
    left: 308px;
    bottom: 1188px;
    border-radius: 20px;
}