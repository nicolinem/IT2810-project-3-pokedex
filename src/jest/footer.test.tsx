import { expect } from "@jest/globals";
import renderer from "react-test-renderer";
import Footer from "../common/components/Footer";

describe("Footer", () => {
  it("snapshot", () => {
    const tree = renderer.create(<Footer></Footer>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
