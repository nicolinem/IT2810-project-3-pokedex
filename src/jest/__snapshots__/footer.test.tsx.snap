// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<footer
  className="bg-[#121A36] text-white p-4 shadow md:px-6 md:py-8 bg-skin-secondary"
>
  <div
    className="sm:flex sm:items-center sm:justify-between"
  >
    <a
      className="flex items-center mb-4 sm:mb-0"
    >
      <img
        alt="Logo"
        className="h-8 mr-3"
        src="programming.png"
      />
      <span
        className="self-center text-2xl font-semibold whitespace-nowrap text-skin-primary"
      >
        Team 1
      </span>
    </a>
    <ul
      className="flex flex-wrap items-center mb-6 text-sm sm:mb-0 text-skin-primary"
    >
      <li>
        <a
          className="mr-4 hover:underline md:mr-6 "
          href="https://www.ntnu.no/studier/emner/IT2810#tab=omEmnet"
        >
          About
        </a>
      </li>
      <li>
        <a
          className="mr-4 hover:underline md:mr-6 "
          href="https://gitlab.stud.idi.ntnu.no/it2810-h22/Team-1/project-3"
        >
          GitLab
        </a>
      </li>
      <li>
        <a
          className="hover:underline"
          href="https://www.ntnu.no/idi"
        >
          Contact
        </a>
      </li>
    </ul>
  </div>
  <hr
    className="my-6 text-skin-primary sm:mx-auto lg:my-8"
  />
  <span
    className="block text-sm sm:text-center text-skin-primary"
  >
    © 2022
    <p>
       
    </p>
    <a
      className="hover:underline"
      href="https://www.ntnu.no/"
    >
      Women in Tech NTNU™
    </a>
    . All Rights Reserved.
  </span>
</footer>
`;
